source/common/tracing/http_tracer_impl.cc

```diff
@@ -40,6 +25,20 @@ static std::string valueOrDefault(const Http::HeaderEntry* header, const char* d
   return header ? header->value().c_str() : default_value;
 }
 
+static std::string buildUrl(const Http::HeaderMap& request_headers) {
+  std::string path = request_headers.EnvoyOriginalPath()
+                         ? request_headers.EnvoyOriginalPath()->value().c_str()
+                         : request_headers.Path()->value().c_str();
+  std::string url = fmt::format("{}://{}{}", valueOrDefault(request_headers.Scheme(), ""),
```

Comment:
The same code is being used to create the `http.url` for both the client and server spans, so does it need to get the scheme from different places depending on whether dealing with an inbound vs outbound request?